static/app/components/events/interfaces/threads.tsx

```diff
@@ -85,6 +81,22 @@ export function getThreadStateIcon(state: ThreadStates | undefined) {
       return <IconInfo />;
   }
 }
+// We want to track the active thread but that can change if the eventId changes, so if the eventID we set the state to be the new best thread
```

Comment:
> so if the eventID we set

Missed a word! Also might want to change the wording to be more precise. We want to reset the active thread every time the event changes because we can't assume they are the same between events.